website/docs/plugin/framework/migrating/attributes-blocks/default-values.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-page_title: 'Attribute default values: Migrating from SDKv2 to the Framework'
+page_title: Migrating attribute default values
 description: >-
-  Specify a default when the Terraform configuration does not supply a value for resource attributes.
-  Migrate attribute defaults in SDKv2 to AttributePlanModifier in the Framework.
+  Learn how to migrate attribute default values from SDKv2 by using an
+  AttributePlanModifier in the Framework.
```

Comment:
This type name is just outdated in the original docs, we can just refer to it like this:

```suggestion
  attribute plan modifier in the Framework.
```